src/rabbit_core_ff.erl

```diff
@@ -95,3 +103,19 @@ remove_explicit_default_bindings(FeatureName, Queues) ->
     [rabbit_binding:remove_default_exchange_binding_rows_of(Q)
      || Q <- Queues],
     ok.
+
+
+%% -------------------------------------------------------------------
+%% Virtual host metadata.
+%% -------------------------------------------------------------------
+
+virtual_host_metadata_migration(_FeatureName, _FeatureProps, enable) ->
+  Tab = rabbit_vhost,
+  rabbit_table:wait([Tab]),
+  Fun = fun vhost:upgrade/1,
+  case mnesia:transform_table(Tab, Fun, [virtual_host, limits, metadata]) of
```

Comment:
Please use something like `amqqueue:fields(amqqueue_v2)` to query the list of fields, instead of hard-coding it.